css/cta.css

```diff
@@ -1,295 +1,130 @@
-padding: 0px;
-gap: 48px;
-
-position: absolute;
-width: 540px;
-height: 356px;
-left: 165px;
-top: calc(50% - 356px/2 + 0.5px);
+.iphoneX2 {
+    position: absolute;
+    width: 230px;
+    height: 465px;
 }
 
-.ctamessage {
-/* CTA Message */
-
-
-/* Auto layout */
-
-display: flex;
-flex-direction: column;
-align-items: flex-start;
-padding: 0px;
-gap: 8px;
-
-width: 540px;
-height: 232px;
-
-
-/* Inside auto layout */
-
-flex: none;
-order: 0;
-align-self: center;
-flex-grow: 0;
+.phoneMockup1 {
+    position: absolute;
+    width: 230px;
+    height: 465px;
+    top: -15px;
+    left: 780px;
 }
 
-.ctasectionheading {
-/* Section Heading */
-
-
-/* Auto layout */
-
-display: flex;
-flex-direction: column;
-align-items: flex-start;
-padding: 0px;
-gap: 8px;
-
-width: 540px;
-height: 232px;
-
-
-/* Inside auto layout */
-
-flex: none;
-order: 0;
-flex-grow: 0;
+.ctaScreen1 {
+    position: absolute;
+    width: 200px;
+    height: 440px;
+    left: 15px;
+    top: 15px;
 }
 
-.ctasectionheading2 {
-/* Section Heading */
-
-
-/* Auto layout */
-
-display: flex;
-flex-direction: column;
-align-items: flex-start;
-padding: 0px;
-gap: 8px;
-
-width: 540px;
-height: 128px;
-
-
-/* Inside auto layout */
-
-flex: none;
-order: 0;
-flex-grow: 0;
+.iphoneX1 {
+    position: absolute;
+    width: 230px;
+    height: 465px;
+    top: 15px;
 }
 
-h2 {
-/* Section Title */
-
-
-width: 540px;
-height: 128px;
-
-/* Heading / H2 */
-
-font-family: 'Inter';
-font-style: normal;
-font-weight: 800;
-font-size: 48px;
-line-height: 64px;
-/* or 133% */
-
-font-feature-settings: 'salt' on, 'liga' off;
-
-/* Text / Gray 900 */
-
-color: #18191F;
-
-
-/* Inside auto layout */
-
-flex: none;
-order: 0;
-flex-grow: 0;
+.content {
+    position: absolute;
+    width: 540px;
+    height: 356px;
+    left: 165px;
+    top: 70px;
+    gap: 50px;
 }
 
-p {
-/* Description */
-
-width: 540px;
-height: 96px;
-
-/* Lead / Lead 1 */
-
-font-family: 'Inter';
-font-style: normal;
-font-weight: 400;
-font-size: 18px;
-line-height: 32px;
-/* or 178% */
-
-font-feature-settings: 'salt' on, 'liga' off;
-
-/* Text / Gray 900 */
-
-color: #18191F;
-
-
-/* Inside auto layout */
-
-flex: none;
-order: 1;
-align-self: stretch;
-flex-grow: 0;
+.ctaMessage {
+    width: 540px;
+    height: 230px;
+    gap: 10px;
 }
 
-.gettheapp {
-    /* Get the App */
-    /* Auto layout */
-
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    padding: 0px;
-    gap: 8px;
-
-    width: 282px;
-    height: 76px;
-
-
-    /* Inside auto layout */
-
-    flex: none;
-    order: 1;
-    align-self: flex-start;
-    flex-grow: 0;
+.sectionHeading {
+    width: 540px;
+    height: 130px;
+    font-family: 'Inter';
+    font-weight: 800;
+    font-size: 50px;
+    line-height: 65px;
 }
 
-.label {
-    width: 104px;
-    height: 28px;
+.description {
+    position: absolute;
+    width: 540px;
+    height: 95px;
+    top: 135px;
+    font-family: 'Inter';
```

Comment:
you can use the font from https://fonts.google.com/  
search exact font from there and use it as a link .